diff --git a/dist/src/stryker-setup.js b/dist/src/stryker-setup.js
index aecfacbe47b285c272c47e54b3d66a716adfdc41..bb3c3bdc331baaa3e0858d9464d9b65c83e126a5 100644
--- a/dist/src/stryker-setup.js
+++ b/dist/src/stryker-setup.js
@@ -4,6 +4,7 @@ import { beforeEach, afterAll, beforeAll, afterEach, inject, } from 'vitest';
 const globalNamespace = inject('globalNamespace');
 const mutantActivation = inject('mutantActivation');
 const mode = inject('mode');
+const testNameSeparator = inject('testNameSeparator');
 const ns = globalThis[globalNamespace] || (globalThis[globalNamespace] = {});
 ns.hitLimit = inject('hitLimit');
 const isGreaterThanVitest4Point1 = inject('isGreaterThanVitest4Point1');
@@ -63,7 +64,7 @@ function collectTestName({ name, suite, }) {
         nameParts.unshift(currentSuite.name);
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(testNameSeparator).trim();
 }
 function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
diff --git a/dist/src/test-helpers.js b/dist/src/test-helpers.js
index 56cc786f30c524ae18ce87eef1a67fdf26b703c5..ad0eff4525c274f394470b99238de85bbae9134c 100644
--- a/dist/src/test-helpers.js
+++ b/dist/src/test-helpers.js
@@ -1,16 +1,16 @@
 // Don't merge this file into 'vitest-helpers.ts'!
 // This file is used from the testing environment (via stryker-setup.js) and thus could be loaded into the browser (when using vitest with browser mode).
 // Thus we should avoid unnecessary dependencies in this file.
-export function collectTestName({ name, suite, }) {
+export function collectTestName({ name, suite, }, separator = ' ') {
     const nameParts = [name];
     let currentSuite = suite;
     while (currentSuite) {
         nameParts.unshift(currentSuite.name);
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(separator).trim();
 }
-export function toRawTestId(test) {
-    return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
+export function toRawTestId(test, separator = ' ') {
+    return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test, separator)}`;
 }
 //# sourceMappingURL=test-helpers.js.map
\ No newline at end of file
diff --git a/dist/src/vitest-helpers.js b/dist/src/vitest-helpers.js
index fa4f8138ab3b2aba64f4e715be2d8126da6ccf8b..1b7ede0e76b017705bb8ed073f316825cb89ad0f 100644
--- a/dist/src/vitest-helpers.js
+++ b/dist/src/vitest-helpers.js
@@ -17,11 +17,11 @@ function convertTaskStateToTestStatus(taskState, testMode) {
     }
     return TestStatus.Failed;
 }
-export function convertTestToTestResult(test) {
+export function convertTestToTestResult(test, separator = ' ') {
     const status = convertTaskStateToTestStatus(test.result?.state, test.mode);
     const baseTestResult = {
-        id: normalizeTestId(toRawTestId(test)),
-        name: collectTestName(test),
+        id: normalizeTestId(toRawTestId(test, separator)),
+        name: collectTestName(test, separator),
         timeSpentMs: test.result?.duration ?? 0,
         fileName: test.file?.filepath && path.resolve(test.file.filepath),
     };
diff --git a/dist/src/vitest-test-runner.js b/dist/src/vitest-test-runner.js
index 84f1ca026503b8e92b7adc952ec6c6a401943656..e1c3553b1bdcecd6bdd0db1b7ebc360caad955ef 100644
--- a/dist/src/vitest-test-runner.js
+++ b/dist/src/vitest-test-runner.js
@@ -18,6 +18,7 @@ export class VitestTestRunner {
         'globalNamespace',
     ];
     ctx;
+    testNameSeparator = ' ';
     options;
     localSetupFile = path.resolve(`./stryker-setup-${process.env.STRYKER_MUTATOR_WORKER ?? 0}.js`);
     constructor(options, log, globalNamespace) {
@@ -81,6 +82,8 @@ export class VitestTestRunner {
             bail: this.options.disableBail ? 0 : 1,
             onConsoleLog: () => false,
         });
+        this.testNameSeparator = semver.major(vitestWrapper.version) >= 5 ? ' > ' : ' ';
+        this.ctx.provide('testNameSeparator', this.testNameSeparator);
         this.ctx.provide('globalNamespace', this.globalNamespace);
         this.ctx.provide('isGreaterThanVitest4Point1', semver.satisfies(vitestWrapper.version, '>=4.1.0'));
         this.ctx.config.browser.screenshotFailures = false;
@@ -173,7 +176,7 @@ export class VitestTestRunner {
             .filter((test) => test.result); // if no result: it was skipped because of bail
         let failure = false;
         const testResults = tests.map((test) => {
-            const testResult = convertTestToTestResult(test);
+            const testResult = convertTestToTestResult(test, this.testNameSeparator);
             failure ||= testResult.status === TestStatus.Failed;
             return testResult;
         });
